client/src/app/components/forms/rsvp/rsvp.component.ts

```diff
@@ -151,7 +164,7 @@ export class RsvpFormComponent implements OnChanges {
 
 			guestFormGroup.controls['firstName'].setValue(firstName);
 			guestFormGroup.controls['lastName'].setValue(lastName);
-			guestFormGroup.controls['attending'].setValue(isGoing);
+			guestFormGroup.controls['attending']?.setValue(isGoing);
```

Comment:
why optional?

Reply:
The associates don't have this item so having it be mandatory breaks autofill

Reply:
don’t we need this to be mandatory, though?

Reply:
so they can’t just submit without saying if they’re going

Reply:
It is mandatory for the main user, this is just autofill

Reply:
k